lib/v2/apiseven/blog.js

```diff
@@ -0,0 +1,50 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+function getDetail(link, ctx) {
+    return ctx.cache.tryGet(link, async () => {
+        const { data: res } = await got(link);
+        const $ = cheerio.load(res);
+        const json = JSON.parse($('#__NEXT_DATA__').text());
+        return {
+            author: json.props.pageProps.post.author_name,
```

Comment:
description is also presented in `json.props.pageProps.post.content`, please add it as well,